src/styles/_catalog.scss

```diff
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform 300ms linear, color 300ms;
+
+    &:hover {
+      cursor: pointer;
```

Comment:
move `cursor: pointer;` to  `&__card`. Don't use this property only for hover effect